homeassistant/components/binary_sensor/hydrawise.py

```diff
@@ -0,0 +1,87 @@
+"""
+Support for Hydrawise sprinkler.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.hydrawise/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.hydrawise import (
+    BINARY_SENSORS, DATA_HYDRAWISE, HydrawiseEntity, ICON_MAP)
+from homeassistant.components.binary_sensor import (
+    BinarySensorDevice, PLATFORM_SCHEMA)
+from homeassistant.const import CONF_MONITORED_CONDITIONS
+
+DEPENDENCIES = ['hydrawise']
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=list(BINARY_SENSORS)):
```

Comment:
`BINARY_SENSORS` is a list: `BINARY_SENSORS = ['is_watering', 'status', 'rain_sensor']`

Reply:
Fixed.